.github/CHANGELOG.md

```diff
@@ -2,6 +2,66 @@
 
 <h3>New features since last release</h3>
 
+* Adds a decorator `@qml.qfunc_transform` to easily create a transformation
+  that transforms a quantum function.
```

Comment:
```suggestion
  that modifies the behaviour of a quantum function.
```
Or something to this effect, just to be a bit more specific about what a transform is actually doing.